src/main/java/httpApplication/SimpleApplication.java

```diff
@@ -0,0 +1,18 @@
+package httpApplication;
+
+import httpserver.HttpIncomingMessage;
+import httpserver.HttpResponse;
+
+public class SimpleApplication implements Application{
```

Comment:
Excellent. This is a great example of a generic name for an abstraction, and a more specific name for the concretion. I would like to see more of this 😄